src/extension/background-script/actions/nostr/helpers.ts

```diff
@@ -58,14 +58,14 @@ export async function addPermissionFor(method: string, host: string) {
   return !!permissionIsAdded && (await db.saveToStorage());
 }
 
-// from: https://github.com/nbd-wtf/nostr-tools/blob/160987472fd4922dd80c75648ca8939dd2d96cc0/event.ts#L61
+// from: https://github.com/nbd-wtf/nostr-tools/blob/b9a7f814aaa08a4b1cec705517b664390abd3f69/event.ts#L95
 // to avoid the additional dependency
 export function validateEvent(event: Event): boolean {
+  if (!(event instanceof Object)) return false;
```

Comment:
We now have multiple changes from the above link, so maybe we should update the comment to say "based upon"?